run_tests.sh

```diff
@@ -50,7 +50,7 @@ fi
 
 # Run unit tests
 echo "Running unit tests:"
-pytest -vvv -s --cov-report term --cov --color=yes --durations=1
+export $(xargs < conf/manager.env) && pytest -vvv -s --cov-report term --cov --color=yes --durations=1
```

Comment:
Let's not do this please. Unit tests should not be dependent on environment. You can set the environment variables inside the unit tests. Check `test/conftest.py` for more and add it there please.